sdk/spring-sdk-testkit/src/main/java/kalix/springsdk/testkit/ActionTestkit.java

```diff
@@ -75,6 +75,23 @@ public <R> ActionResult<R> call(Function<A, Action.Effect<R>> func) {
     return new ActionResultImpl<>(func.apply(createAction(context)));
   }
 
+  /**
+   * The {@code call} method can be used to simulate a unary call to the Action. The passed java lambda should
+   * return an Action.Effect. The Effect is interpreted into an ActionResult that can be used in
+   * test assertions.
+   *
+   * @param func A function from Action to Action.Effect
+   * @param metadata Medatada accessible to the Action
+   * @return an ActionResult
+   * @param <R> The type of reply that is expected from invoking a command handler
+   */
+  public <R> ActionResult<R> call(Function<A, Action.Effect<R>> func, Metadata metadata) {
```

Comment:
I found we will need to create such a call to pass the subject, at least. To allow retrieval of the value [here](https://github.com/lightbend/kalix-jvm-sdk/pull/1264/files#diff-00139a7ecdb8f7994ec8b144ed5fb581be8f45060a59c7f8bfd2e30a02c78f61R24)